src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/K8sServiceSpecForms/KubernetesServiceSpecEditable.tsx

```diff
@@ -20,6 +21,25 @@ import { setupMode } from '../K8sServiceSpecHelper'
 import type { KubernetesServiceInputFormProps } from '../K8sServiceSpecInterface'
 import css from '../K8sServiceSpec.module.scss'
 
+const getManifestCardHeader = (
+  selectedDeploymentType: string,
+  getString: (key: keyof StringsMap, vars?: Record<string, any> | undefined) => string
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232297)
```suggestion
  getString: (key: keyof StringsMap, vars?: Record<string, unknown> | undefined) => string
```